src/docfx/config/AppData.cs

```diff
@@ -25,6 +25,10 @@ internal static class AppData
 
         public static string DefaultGitHubUserCachePath => Path.Combine(CacheRoot, "github-users.json");
 
+        public static string RepositoryUrl => GetRepositoryUrlFromEnvironment();
```

Comment:
`AppData` is where we manage the content inside `.docfx` folder, you can read environment variables directly inside `Repository`.